### DISCUSSION TITLE
Create DatabaseConnection from sqlx::mysql::MySqlConnectOptions

### DISCUSSION THREAD

**Discussion by cemoktra** [SeaQL/sea-orm#392] [Q&A] [answered]
Hi,

it would be nice to create a connection from the underlying sql options. It offers more configuration and is easier to use than the connection string

**Comment by billy1624** [comment]
Hey @cemoktra, yes, you could.
Configure and create a `Pool` on your own then supply it to sea-orm.
https://docs.rs/sea-orm/0.4.2/sea_orm/?search=from_sqlx_

**Comment by cemoktra** [comment, marked as answer]
To understand this right

- create `MySqlConnectOptions`
- create `MysqlPool` from `MySqlConnectOptions` using `connect_with` 
- use `from_sqlx_mysql_pool` in sea orm

**Comment by tyt2y3** [comment]
Yes you are right

**Comment by cemoktra** [comment]
Might be a nice addition to the docs as i was fighting ssl options for quite some time today
